Release 0.32.0 (#961)
fluxcd/source-controller: CHANGELOG.md

All notable changes to this project are documented in this file.

## 0.32.0

**Release date:** 2022-11-17

> This prerelease comes with a major refactoring of its Git implementation.

Reviewer: ```suggestion
This prerelease comes with a major refactoring of the controller's Git operations.
```
